src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,100 @@ title = "Releases"
+  instead of kubeconfigs.
+  * Existing options (`--kubecontext` etc.) are preserved for backwards
+  compatibility, but are deprecated and will be removed in the next
+  release.
+
+### Other changes
+
+* The flannel CNI is now properly identified during join.
+* A new ServiceExport status condition type named Synced was added that indicates whether or not the ServiceImport
+was successfully synced to the broker.
+* Service Discovery now handles updates to an exported service and updates/deletes the corresponding ServiceImport accordingly.
+* Deploying using Shipyard now uses specific image overrides determined by the `PRELOAD_IMAGES` variable.
+For developers using `make deploy` (or `make e2e`) there won't be a noticeable change in behavior, but only those images
+specified by the `PRELOAD_IMAGES` will be preloaded and used. Developers can also specify the variable explicitly to further
+control which images get preloaded and used.
+* Added a centralized script for getting `subctl`. It now impersonates `subctl` so that users who run `make shell` will have it
+downloaded the first time they try to use it. To influence the version being installed, the `SUBCTL_VERSION` variable can be set.
+* All `shflags` usage has been removed from Shipyard and consuming projects.
+Users/Developers running commands with `using=` are not affected.
+Users/Developers running commands with `*_ARGS=--flag` should switch to using environment variables instead.
+For example, if you used to run `make deploy DEPLOY_ARGS='--settings ...'`, please run `make deploy SETTINGS='...'` instead.
+* If it’s not given a custom directory, `subctl gather` stores all its output in a directory
+named `submariner-` followed by the current date and time (in UTC) in "YYYYMMDDHHmmss" format.
+* Service Discovery now returns a DNS error message in the response body when no matching records are found for the query to
+`clusterset.local`. This prevents unnecessary retries.
+* Cloud cleanup for OpenStack now identifies and deletes failed machinesets.
+* Privileges of the Route Agent and Gateway pods were reduced as they don’t need to access persistentvolumeclaims and secrets.
+be used to verify if `submariner_router` is pinned to the proper Gateway node.
+* The privileged SCC permission for Submariner components in OCP is set now by creating separate `ClusterRole` and `ClusterRoleBinding`
+resources instead of manipulating the system privileged SCC resource.
+* It is now possible to customize the default TCP MSS clamping value set by Submariner also for non-Globalnet deployments.
+* The `subctl show` command now correctly reports component image versions when image overrides were specified on `join`.
+* The `subctl gather` command now creates one subdirectory per cluster instead of embedding the cluster name in each file name.
+* The `subctl gather` command now includes the output from `ovn-sbctl show` which has the chassis-id to hostname mapping that can
```

Comment:
```suggestion
* The `subctl gather` command now includes the output from `ovn-sbctl show` which has the `chassis-id` to `hostname` mapping that can
```
If these are field names, they should be `code case`.

https://kubernetes.io/docs/contribute/style/style-guide/#use-code-style-for-object-field-names-and-namespaces